apps/common-app/package.json

```diff
@@ -57,5 +57,8 @@
     "react-native-screens": "3.31.0-rc.1",
     "react-native-svg": "^15.2.0-rc.0",
     "typescript": "~5.3.0"
+  },
```

Comment:
ok, but i'm curious what is the reason why? 🤔

Reply:
It's supposed only to be a container for the common javascript code. I want to avoid the case where its dependencies would be pulled into the example apps and potentially duplicate some libraries. We only have those libraries are dev dependencies for TypeScript here.

Reply:
I remember the pain in the previous repo where app and ExampleApp had different versions of some package 💀